### DISCUSSION TITLE
Laser Loader Flashing

### DISCUSSION THREAD

**Discussion by ConnorsSilverpaws** [FakeFishGames/Barotrauma#13987] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Attempting to wire a pulse laser loader causes a red flash light as if it was firing. Tested on Barsuk
![pulse laser loader flash](https://github.com/FakeFishGames/Barotrauma/assets/39156231/90bf6c69-35c7-4e74-b7a5-5310b647e079)

### Reproduction steps

1. Hold screwdriver in hand
2. Press E on Pulse Laser Loader
3. See it flash red
4. Move the wiring UI out of the way
5. Spam E on Pulse Laser Loader for fun

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/5ac30e6149e1b9c71962602ff51bd15fdc03e91e (commit in our private repo).
